tests/cupy_tests/core_tests/test_raw.py

```diff
@@ -341,12 +342,21 @@ def use_temporary_cache_dir():
                 yield path
 
 
-@testing.parameterize(*testing.product({
-    'backend': ('nvrtc', 'nvcc'),
-}))
+_in_memory_cache = os.environ.get('CUPY_CACHE_IN_MEMORY')
+
+
+@testing.parameterize(
+    {'backend': 'nvrtc', 'in_memory': False},
+    {'backend': 'nvrtc', 'in_memory': True},
+    {'backend': 'nvcc', 'in_memory': False},
+)
 class TestRaw(unittest.TestCase):
 
     def setUp(self):
+        if self.backend == 'nvcc' and self.in_memory == '1':
+            self.skipTest('nvcc does not support in-memory cache')
```

Comment:
This parameter combination is not listed now.

Reply:
Sorry I was still working on it, but got interrupted due to power outage (our area is very unstable lately). Will push once I get the chance.